ibm_catalog.json

```diff
@@ -0,0 +1,211 @@
+        "terraform",
+        "solution"
+      ],
+      "provider_name": "IBM",
+      "short_description": "Deploys IBM Monitoring Agent to a cluster",
+      "long_description": "Solutions that support deploying IBM Monitoring Agent for logging.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-monitoring-agent/blob/main/solutions/fully-configurable/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-monitoring-agent/main/images/monitoring-icon.svg",
+      "flavors": [
+        {
+          "label": "Fully Configurable",
+          "name": "fully-configurable",
+          "working_directory": "solutions/fully-configurable",
+          "compliance": {},
+          "iam_permissions": [
+            {
+              "service_name": "containers-kubernetes",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Viewer"
+              ]
+            }
+          ],
+          "architecture": {
+            "features": [
+              {
+                "title": "Works with any Openshift or k8s cluster",
+                "description": "Yes"
+              },
+              {
+                "title": "Deploys the IBM Monitoring Agent on an existing cluster.",
+                "description": "Yes"
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-monitoring-agent/tree/main/reference-architecture/deployable-architecture-monitoring-agent.svg",
+                  "caption": "Monitoring Agent on a cluster",
+                  "type": "image/svg+xml"
+                },
+                "description": "Monitoring Agent deployed on a cluster."
+              }
+            ]
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "required": true
+            },
+            {
+              "key": "cluster_id",
+              "custom_config": {
+                  "type": "cluster_var",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment"
+              },
+              "required": true
+            },
+            {
+              "key": "cluster_resource_group_id",
+              "custom_config": {
+                  "type": "resource_group",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment",
+                  "config_constraints": {
+                      "identifier": "rg_id"
+                  }
+              },
+              "required": true
+            },
+            {
+              "key": "access_key",
+              "required": true
+            },
+            {
+              "key": "image_registry",
+              "required": true
```

Comment:
No need to force this as required - it has a default value so can live in the optional tab